### PR TITLE
created HTML for our project

### PR DESCRIPTION
**Pull request by mzubtsova**
-added navigation section for hamburger menu;
-added header and form for app
-added div with recipe results for suture styling

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,28 @@
...
+        <div class="container initial">
+            <h1 class="headerText">Recipe App</h1>
+            <form id="form">
+                <input name="recipeName" type="text" id="text" placeholder="Search Your Recipe..." value="">
+                <button>Search</button>
+            </form>
+            <div class="searchResult">
+                <ul class="recipes"></ul>
+            </div>
+        </div>
```
Comment (shradhaMe) [inline index.html:31]: This looks great. Just you missed section closing tag.

**Review by shradhaMe — CHANGES_REQUESTED** [review]
Looks good, just one thing please close your section.

File: index.html (hunk 1 above)
Comment (mzubtsova) [reply index.html:31]: Added closing tag for section.

**Comment by mzubtsova** [comment]
Made requested changes

**Review by shradhaMe — APPROVED** [review]
We are all good. Go ahead and merge.

**Pull request merged**